src/injected/is-visualization-enabled.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { TestMode } from 'common/configs/test-mode';
+import { VisualizationConfiguration } from 'common/configs/visualization-configuration';
+import { AssessmentStoreData } from 'common/types/store-data/assessment-result-data';
+import { TabStoreData } from 'common/types/store-data/tab-store-data';
+import { VisualizationStoreData } from 'common/types/store-data/visualization-store-data';
+
+export type IsVisualizationEnabledCallback = (
+    config: VisualizationConfiguration,
+    step: string,
+    visualizationState: VisualizationStoreData,
+    assessmentState: AssessmentStoreData,
+    tabState: TabStoreData,
+) => boolean;
+
+export const isVisualizationEnabled: IsVisualizationEnabledCallback = (
+    config: VisualizationConfiguration,
+    step: string,
+    visualizationState: VisualizationStoreData,
+    assessmentState: AssessmentStoreData,
+    tabState: TabStoreData,
+) => {
+    const scanData = config.getStoreData(visualizationState.tests);
+    return (
+        config.getTestStatus(scanData, step) &&
```

Comment:
**not for this PR** but since I haven't work on the visualization/drawing logic in a while... I found `config.getTestStatus` name confusing... is returning a boolean, I'm guessing this has to do with the test being enabled or not, which, if true, points to the proper name for this function.